apps/nextra/pages/en/build/indexer/api/labs-hosted.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "Labs-Hosted Indexer API"
+---
+
+# Labs-Hosted Indexer API
+
+import { IndexerBetaNotice } from '@components/index';
+
+<IndexerBetaNotice />
+
+## GraphQL API Endpoints
+
+When making GraphQL queries to the Labs-Hosted Indexer API, use the following endpoints:
+
+- **Mainnet:** https://api.mainnet.aptoslabs.com/v1/graphql
+- **Testnet:** https://api.testnet.aptoslabs.com/v1/graphql
+- **Devnet:** https://api.devnet.aptoslabs.com/v1/graphql
```

Comment:
Also, I think these links aren't meant to be clicked - do you think it's alright if I wrap them in codeblocks? @hariria 

This is what that'd look like:
<img width="868" alt="image" src="https://github.com/aptos-labs/developer-docs/assets/163934542/d0cbf596-9d2b-46be-bad1-440287884db2">

Reply:
You're correct, those are not, but my comment was on lines 23-25 which are different from the links above. The `https://cloud.hasura.io` links open up the graphql explorer for indexer

Reply:
I added a screenshot of what the cards would look like here :) - https://github.com/aptos-labs/developer-docs/pull/285#discussion_r1578548164